learn/getting_started/quick_start.md

```diff
@@ -279,6 +279,10 @@ These allow you to configure your Meilisearch instance at launch with [environme
 
 **Instance options affect your entire Meilisearch instance**, not just a single index.
 
+#### Configuration file
+
+Instead of providing command-line options or environment variables, you can provide a `.toml` [configuration file](/learn/configuration/instance_options.md#configuration-file). Once downloaded, move it to your working directory and it will be loaded at launch. Like instance options, it affects your entire instance.
```

Comment:
Trying to avoid repeating the same verb twice in quick succession.

```suggestion
Instead of providing command-line options or environment variables, you can use a `.toml` [configuration file](/learn/configuration/instance_options.md#configuration-file). Once downloaded, move it to your working directory and it will be loaded at launch. Like instance options, it affects your entire instance.
```